bin/yaml2make

```diff
@@ -70,6 +70,11 @@ def read_file(test, type, run_index):
     matches = [os.path.join(TOPDIR, p, test, type) for p in TEST_PATHS 
                 if os.path.exists(os.path.join(TOPDIR, p, test, type))]
 
+    # Allow the user to specify full path to test folder
```

Comment:
Where would these test folders exist?

Reply:
Does this imply there are test-programs being used on CV32E40X that are **_not_** in this repo?

Reply:
The tests for CV32E40X are placed in this or the cv32e40x-dv repo, this change will make it possible to  for instance build temporary test programs for debugging purposes without having to place them into a hardcoded path.